force-app/main/default/reports/ConstructionReportsFolder.reportFolder-meta.xml

```diff
@@ -2,7 +2,7 @@
 <ReportFolder xmlns="http://soap.sforce.com/2006/04/metadata">
     <folderShares>
         <accessLevel>Manage</accessLevel>
-        <sharedTo>test-mzyso6wots4v@example.com</sharedTo><!-- Change to admin username in the org-->
```

Comment:
run the git update-index --assume-unchanged command for this file so we don't have to have this in our commits anymore